g4f/Provider/bing/create_images.py

```diff
@@ -194,48 +178,47 @@ def get_cookies_from_browser(proxy: str = None) -> Dict[str, str]:
 
 class CreateImagesBing:
     """A class for creating images using Bing."""
-
-    _cookies: Dict[str, str] = {}
     
-    @classmethod
-    def create_completion(cls, prompt: str, cookies: Dict[str, str] = None, proxy: str = None) -> Generator[str, None, None]:
+    def __init__(self, cookies: dict[str, str] = {}, proxy: str = None) -> None:
+        self.cookies = cookies
+        self.proxy = proxy
+    
+    def create_completion(self, prompt: str) -> Generator[ImageResponse, None, None]:
         """
         Generator for creating imagecompletion based on a prompt.
 
         Args:
             prompt (str): Prompt to generate images.
-            cookies (Dict[str, str], optional): Cookies for the session. If None, cookies are retrieved automatically.
-            proxy (str, optional): Proxy configuration.
 
         Yields:
             Generator[str, None, None]: The final output as markdown formatted string with images.
         """
-        loop = get_event_loop()
-        cookies = cookies or cls._cookies or get_cookies(".bing.com")
+        cookies = self.cookies or get_cookies(".bing.com")
         if "_U" not in cookies:
             login_url = os.environ.get("G4F_LOGIN_URL")
             if login_url:
                 yield f"Please login: [Bing]({login_url})\n\n"
-            cls._cookies = cookies = get_cookies_from_browser(proxy)
-        yield loop.run_until_complete(create_images_markdown(cookies, prompt, proxy))
+            self.cookies = get_cookies_from_browser(self.proxy)
+        yield asyncio.run(self.create_async(prompt))
 
-    @classmethod
-    async def create_async(cls, prompt: str, cookies: Dict[str, str] = None, proxy: str = None) -> str:
+    async def create_async(self, prompt: str) -> ImageResponse:
```

Comment:
Consider adding type hints to the `prompt` parameter of the `create_async` method.